core/src/com/unciv/ui/CivilopediaScreen.kt

```diff
@@ -88,11 +88,9 @@ class CivilopediaScreen(ruleset: Ruleset) : CameraStageBaseScreen() {
                 .map { CivilopediaEntry(it.name,it.getDescription(ruleset),
                         ImageGetter.getTechIconGroup(it.name,50f)) }
 
-        categoryToEntries["Tutorials"] = Tutorials().getTutorialsOfLanguage("English").keys
-                .filter { !it.startsWith("_") }
-                .map { CivilopediaEntry(it.replace("_"," "),
-                        Tutorials().getTutorials(it, UncivGame.Current.settings.language)
-                                .joinToString("\n\n")) }
+        tutorialMiner.getCivilopediaTutorials(UncivGame.Current.settings.language)
+                ?.map { CivilopediaEntry(it.key.value.replace("_"," "), it.value.joinToString("\n\n")) }
+                ?.let { categoryToEntries.put("Tutorials", it) }
```

Comment:
Please do not use .let{}, obviously if we don't drag the previous nullability around this will be .foreach or something